stress_test/controller_test_basic.py

```diff
@@ -11,100 +11,142 @@
-    path_file = os.path.join(datastore_conf_path,'org.opendaylight.controller.cluster.datastore.cfg')
+    '''check the effect of disable_ persistence'''
+    path_file = os.path.join(datastore_conf_path, 'org.opendaylight.'
+                             'controller.cluster.datastore.cfg')
     pattern = 'persistent=false'
 
     cmd = ('grep {0} {1}'.format(pattern, path_file))
-    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn, cmd,'Check_persistence')
-
-    if (output!= None):
-        logging.info ("[Testing] Persistence is disabled successfully")
+    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn,
+                                                       cmd,
+                                                       'Check_persistence')
+    if (output is not None):
+        logging.info('[Testing] Persistence is '
+                     'disabled successfully')
     else:
-        logging.info ("[Testing] Persistence is still enabled")
+        logging.info('[Testing] Persistence is still enabled')
 
 ctrl.generate_xmls()
-try:
 
-    #start a controller
+try:
+    '''start a controller'''
     ctrl.check_status()
     ctrl.start()
-
-    #change stat period
+    '''change stat period'''
     ctrl.change_stats()
-
-    #check the effect of change_stats()
-    path_file = os.path.join(datastore_conf_path,'opendaylight','karaf','30-statistics-manager.xml')
-    pattern = '<min-request-net-monitor-interval>'+str(ctrl.stat_period_ms[0])+'</min-request-net-monitor-interval>'
+    '''check the effect of change_stats()'''
+    path_file = os.path.join(datastore_conf_path,
+                             'opendaylight',
+                             'karaf',
+                             '30-statistics-manager.xml')
+    pattern = '<min-request-net-monitor-'
+    'interval>' + str(ctrl.stat_period_ms[0]) + '</min-request-net-'
+    'monitor-interval>'
 
     cmd = ('grep {0} {1}'.format(pattern, path_file))
-    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn, cmd,'Check_statistics_period')
-    if (output!= None):
-        logging.info ("[Testing] Interval statistics has been updated successfully")
+    exit_status, output = util.netutil.ssh_run_command(ctrl._ssh_conn,
+                                                       cmd,
+                                                       'Check_stats_period')
+    if (output is not None):
+        logging.info('[Testing] Interval statistics have been '
+                     'updated successfully')
     else:
-        logging.info ("[Testing] Interval statistics not updated")
-
+        logging.info("[Testing] Interval statistics not updated")
+
+    '''get information about Controller Datastore'''
+    try:
+        hosts = ctrl.get_oper_hosts()
+        print('[Testing] The number of hosts on datastore '
+              'are: {0}'.format(hosts))
+    except AttributeError:
+        print('[Testing] Error during the query of hosts in DS')
+    try:
```

Comment:
leave an empty line between consecutive try-except blocks, in order to separate them

do this everywhere